### PR TITLE
Bring some consistency to module descriptions.

### PR DESCRIPTION
**Pull request by jpmedley**
R: @jeffposnick @addyosmani @gauntface

### CODE REVIEW

File: packages/workbox-google-analytics/src/index.js [hunk 1]
```diff
@@ -19,12 +19,10 @@ import logHelper from '../../../lib/log-helper.js';
 import replayQueuedRequests from './lib/replay-queued-requests.js';
 
 /**
- * In order to use the library, call
- * `workbox.googleAnalytics.initialize()`.
- * It will take care of setting up service worker `fetch` handlers to ensure
- * that the Google Analytics JavaScript is available offline, and that any
- * Google Analytics requests made while offline are saved (using `IndexedDB`)
- * and retried the next time the service worker starts up.
+ * A module that ensures Google Analytics works offline. It does this by
```
Comment (addyosmani) [inline packages/workbox-google-analytics/src/index.js]: Would this not be a library? I see `module` as being used for packages that can really only be consumed via Node while browser/client facing ones are a library an end user consumes more generally.

`workbox-precaching` would be a module.

**Comment by jpmedley** [comment]
I've changed these further to better match the language in Jeff's [README PR](https://github.com/GoogleChrome/workbox/pull/554/files).

File: packages/workbox-routing/src/index.js [hunk 2]
```diff
@@ -20,6 +20,23 @@
  *
  * **Install:** `npm install --save-dev workbox-routing`
  *
+ * **Note:** A longer version of this example is available [here](../../examples/workbox-routing/index.html)
```
Comment (gauntface) [inline packages/workbox-routing/src/index.js]: This URL should just be /examples/workbox-routing/

File: packages/workbox-precaching/src/index.js [hunk 3]
```diff
@@ -16,8 +16,10 @@
 /**
  * # workbox-precaching
  *
- * The precaching module provides helpers that make it easy to cache files
- * during the install step of your service worker.
+ * A lover-level service worker library to precache a manifest of URLs.
+ *
+ * It's recommended that you use the higher-level [`workbox-sw`](../workbox-sw)
```
Comment (gauntface) [inline packages/workbox-precaching/src/index.js]: Please don't make relative.

File: packages/workbox-build/src/index.js [hunk 4]
```diff
@@ -7,6 +7,14 @@ const injectManifest = require('./lib/inject-manifest');
 /**
  * # workbox-build
  *
+ * A module that integrates into your build process, helping you generate a
+ * manifest of local files that [`workbox-sw`](../workbox-sw) should precache.
+ * It can also scaffold a full service worker implementation for you.
+ *
+ * While you can use this module directly, there are both
+ * [command-line](../workbox-cli) and [Webpack](../workbox-webpack-plugin)
```
Comment (gauntface) [inline packages/workbox-build/src/index.js]: Not relative again: /examples/workbox-cli/

File: packages/workbox-precaching/src/index.js (hunk 3 above)
Comment (jpmedley) [inline packages/workbox-precaching/src/index.js]: I copied this from Jeff's README edit. 

@jeffposnick @gauntface Please come to agreement.

**Comment by jpmedley** [comment]
@gauntface PTAL

File: packages/workbox-build/src/index.js [hunk 5]
```diff
@@ -7,9 +7,16 @@ const injectManifest = require('./lib/inject-manifest');
 /**
  * # workbox-build
  *
- * This Node module can be used to generate a list of assets that should be
- * precached in a service worker, generating a hash that can be used to
- * intelligently update a cache when the service worker is updated.
+ * A module that integrates into your build process, helping you generate a
+ * manifest of local files that [`workbox-sw`](../workbox-sw) should precache.
```
Comment (gauntface) [inline packages/workbox-build/src/index.js]: This link worries me a little. Where should it be pointing to? If it's JSDoc then please use the expected syntax - of the top of my head, untested, I think: `[workbox-sw]{@link module:workbox-sw}` should work.

File: packages/workbox-build/src/index.js [hunk 6]
```diff
@@ -7,9 +7,16 @@ const injectManifest = require('./lib/inject-manifest');
...
+ * It can also scaffold a full service worker implementation for you.
+ *
+ * While you can use this module directly, there are both
+ * [command-line](/how_tos/sw-cli) and
+ * [Webpack](https://www.npmjs.com/package/workbox-webpack-plugin)
```
Comment (gauntface) [inline packages/workbox-build/src/index.js]: should we link to the getting started guide instead? https://workboxjs.org/get-started/webpack

File: packages/workbox-build/src/index.js (hunk 5 above)
Comment (addyosmani) [inline packages/workbox-build/src/index.js]: Let's update to use the JSDoc syntax as prescribed above.

File: packages/workbox-build/src/index.js (hunk 6 above)
Comment (addyosmani) [inline packages/workbox-build/src/index.js]: For consistency, I'd say let's link to on site content for this so 👍 on starter guide for webpack.

**Comment by addyosmani** [comment]
@jpmedley Friendly ping. If you're short on time we can make the changes requested above directly.

**Pull request closed without merging**